src/factory.ts

```diff
@@ -120,10 +142,12 @@ export async function buildStrategy(
     bumpMinorPreMajor: options.bumpMinorPreMajor,
     bumpPatchForMinorPreMajor: options.bumpPatchForMinorPreMajor,
   });
+  const commitPartial = await getCommitPartialContent(options);
   const changelogNotes = buildChangelogNotes({
     type: options.changelogType || 'default',
     github: options.github,
     changelogSections: options.changelogSections,
+    commitPartial,
```

Comment:
This entire feature is entirely opt-in.

`commitPartial` already existed as as optional argument prior to this PR.
It was just never used.